cmd/neutron_query_relayer/cmd/start.go

```diff
@@ -93,10 +93,20 @@ func startRelayer() {
 		}
 	}(storage)
 
+	subscriber, err := app.NewDefaultSubscriber(cfg, logRegistry)
+	if err != nil {
+		logger.Fatal("failed to create subscriber", zap.Error(err))
+	}
+	relayer, err := app.NewDefaultRelayer(ctx, cfg, logRegistry, storage)
+	if err != nil {
+		logger.Fatal("failed to create relayer", zap.Error(err))
+	}
+	queriesTasksQueue := make(chan neutrontypes.RegisteredQuery, cfg.QueriesTaskQueueCapacity)
+
 	go func() {
 		server := &http.Server{
 			Addr:    fmt.Sprintf(":%d", cfg.WebserverPort),
-			Handler: webserver.Router(logRegistry, storage),
+			Handler: webserver.Router(logRegistry, storage, relayer.GetTxProcessor()),
```

Comment:
So if we pass the txProcessor to another goroutine, we need it to handle concurrency.
I think the only place that we need to guard is TxSender#Send method that uses account sequence.
So how about add mutex lock to TxSender#Send method?

Reply:
its already here.
https://github.com/neutron-org/neutron-query-relayer/blob/main/internal/submit/tx_sender.go#L114